.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
+  - 7.1
+  - 7.2
```

Comment:
Travis does not support `7.2` or `7.3` at the moment 😢 
We'll have to just use `7.1` at the moment